pkg/apis/pipeline/v1/pipelinerun_defaults_test.go

```diff
@@ -25,12 +25,12 @@ import (
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/tektoncd/pipeline/pkg/apis/config"
+	dfttesting "github.com/tektoncd/pipeline/pkg/apis/config/testing"
```

Comment:
nit: cfgtesting might be more clear but that's subjective, will leave it up to you

Reply:
Updated!